### PR TITLE
Feat login signup

### PR DESCRIPTION
**Pull request by ianuragsharma**
Feature added :
Login,signup (without error handling)
image
image

### CODE REVIEW

File: src/features/Auth/components/SignUp.jsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+import { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Link, useNavigate } from "react-router-dom";
+import { FaUserAlt, FaLock, FaUserCircle } from "react-icons/fa";
+import { MdEmail } from "react-icons/md";
+import { signupHandler } from "features";
+const CFaUserAlt = chakra(FaUserAlt);
+const CFaLock = chakra(FaLock);
+const CFMdEmail = chakra(MdEmail);
+const CDFaUserCircle = chakra(FaUserCircle);
```
Comment (arpit01923) [inline src/features/Auth/components/SignUp.jsx:28]: how chakra works for react-icons

**Review by arpit01923 — APPROVED** [review]
- clean UI
- impressive code
- good to merge

**Review by ranitasahaa1230 — APPROVED** [review]
Code can be understood easily 
Clean UI
Looks good to merge

File: src/features/Auth/components/SignUp.jsx (hunk 1 above)
Comment (ianuragsharma) [reply src/features/Auth/components/SignUp.jsx:28]: it works as a wrapper.Gives us the ability to add chakra props .

**Pull request merged**